app/src/main/java/org/wikipedia/readinglist/ReadingListFragment.java

```diff
@@ -599,6 +600,18 @@ void bindItem(ReadingListPage page) {
             getView().setSecondaryActionHint(R.string.reading_list_article_make_offline);
             getView().setSearchQuery(currentSearchQuery);
             PageAvailableOfflineHandler.INSTANCE.check(page, available -> getView().setViewsGreyedOut(!available));
+
+            if (!TextUtils.isEmpty(currentSearchQuery)) {
+                getView().setTitleMaxLines(2);
+                getView().setTitleEllipsis();
+                getView().setDescriptionMaxLines(2);
+                getView().setDescriptionEllipsis();
+                getView().setListItemImageDimensions(DimenUtil.roundedDpToPx(ARTICLE_ITEM_IMAGE_DIMENSION), DimenUtil.roundedDpToPx(ARTICLE_ITEM_IMAGE_DIMENSION));
```

Comment:
The thumbnail size should be set to the default size if out of the actionMode. 
Please see code below: (or you can rebase it)
https://github.com/wikimedia/apps-android-wikipedia/blob/cc25e3fa4e4382983ee0dbef3011145eedaab8ca/app/src/main/java/org/wikipedia/readinglist/ReadingListFragment.java#L610-L628

Reply:
Good catch! thank you..done